catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/InternFormulaStateTest.java

```diff
@@ -71,7 +71,7 @@ public void setUp() throws Exception {
 	@Test
 	public void testEquals() {
 		assertThat(internState, not(equalTo(internStateToCompare)));
-		assertFalse(internState.equals(Integer.valueOf(1)));
+		assertNotEquals(internState, Integer.valueOf(1));
```

Comment:
Here the same. Thank you!

Reply:
I changed them already and noticed that  
`		assertThat(internState, not(equalTo(internStateToCompare)));
`
need to be changed too to be  
`		assertNotEquals(internStateToCompare, internState);
`
I think it's more readable now. What do you think?

Reply:
In my opinion, assertThat is more readable, because it's like a sentence:
Assert that (the) value (is) not equal to (the) unexpected state.
But "assertNotEquals" and "assertEquals" are also readable.
(Mention, it must be "assert(Not)Equals(**(un)expected Value**, **actualValue**).
So, **both** versions are fine ;)

Reply:
Thank you @gPathpp for your review. I've squashed the commits but I don't know why they still 6! 
Do you have any idea?

Reply:
No problem. 
No, I have no idea. 
If the branch is broken or something else, please open a new one and cherry-pick the commit.
Thanks!

Reply:
Thanks, I will try it.